docs/key-issues/gpus.md

```diff
@@ -30,52 +30,66 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
-PyTorch and TensorFlow). For most of its own projects, it uses a
-[Private PyPI Index](https://pypi.org/project/nvidia-pyindex/) - and that also
-includes rebuilds of TensorFlow and other packages.
+If wheels contain CUDA code, they require the CUDA Toolkit (a specific version
+of it at that) to be installed. Again, installers do not know this and there is
+no way to express this dependency. The same will be true for ROCm and other
+types of GPU hardware and languages.
+
+NVIDIA has made steps towards better support for CUDA on PyPI. Various
+library components of the CTK have been packaged as wheels and are now
+distributed on PyPI, such as
+[nvidia-cublas-cu11](https://pypi.org/project/nvidia-cublas-cu11/). Python
+wrappers around CUDA libraries have been consolidated into CUDA Python
+([website](https://developer.nvidia.com/cuda-python), 
+[PyPI package](https://pypi.org/project/cuda-python)), but this package assumes
+that the CUDA runtime and driver are already installed since it only provides
+Python bindings to the API. Many other projects remain hosted on
+[NVIDIA's Private PyPI Index](https://pypi.org/project/nvidia-pyindex/), which
+also includes rebuilds of TensorFlow and other packages.
 
 A single CUDA version supports a reasonable range of GPU architectures. New
 CUDA versions get released regularly, and - because they come with increased
 performance or new functionality - it may be necessary or desirable to build
 new wheels for that CUDA version. If only the supported CUDA version is
 different between two wheels, the wheel tags and filename will be identical.
 Hence it is not possible to upload more than one of those wheels under the same
-package name. To work around that, a project may either support only one CUDA
-version on PyPI, or create different packages. PyTorch and TensorFlow do the
-former, with TensorFlow supporting only a single CUDA version, and PyTorch
-providing more wheels for other CUDA versions and a CPU-only version in a
-separate wheelhouse (see
+package name.
+
+Historically, this required projects to produce packages specific CUDA minor
+versions. Projects would either support only one CUDA version on PyPI, or create
+different packages. PyTorch and TensorFlow do the former, with TensorFlow
+supporting only a single CUDA version, and PyTorch providing more wheels for
+other CUDA versions and a CPU-only version in a separate wheelhouse (see
 [pytorch.org/get-started](https://pytorch.org/get-started/locally/)).
 CuPy provides a number of packages: [`cupy`](https://pypi.org/project/cupy/),
 [`cupy-cuda102`](https://pypi.org/project/cupy-cuda102/),
 [`cupy-cuda110`](https://pypi.org/project/cupy-cuda1110/),
 [`cupy-cuda111`](https://pypi.org/project/cupy-cuda111/),
-[`cupy-cuda11x`](https://pypi.org/project/cupy-cuda11x/),
 [`cupy-rocm-4-3`](https://pypi.org/project/cupy-rocm-4-3/),
 [`cupy-rocm-5-0`](https://pypi.org/project/cupy-rocm-5-0/).
```

Comment:
I am thinking to add a sentence or two here explaining why it's painful to everyone.
```suggestion
[`cupy-rocm-5-0`](https://pypi.org/project/cupy-rocm-5-0/). This not only adds maintenance
overhead to project developers and consumes more storage and network bandwidth for PyPI.org,
but also prevents downstream projects from properly declaring the dependency.
```